dipy/align/__init__.py

```diff
@@ -1,15 +1,18 @@
+from .api import *  # noqa
```

Comment:
We should avoid `import *` and just import the desired function

Reply:
That's why the module has an `__all__` attribute.

Reply:
OK OK. Changed in 274248fa4